Resources/input/fsh/profiles/MessageHeaderMedicationRequest.fsh

```diff
@@ -6,8 +6,12 @@ Description: "Header for Messages concerning Medication Request"
 * insert Meta
 * insert MetaProfile (GEM_PR_ERP_MEDREQ_MessageHeaderMedicationRequest)
 
-* sender 1..
-* sender only Reference(Organization)
-* sender.reference 1..
 
-//Sender and receipient festmachen
\ No newline at end of file
+* event[x] only Coding
+* event[x] from RequestingOrganizationVS
+
+* sender 1..1
+* sender only Reference(KBV_PR_FOR_Organization)
+* sender.reference 1..1
+* source.endpoint 1..1
```

Comment:
Könnte hier festgesetzt werden, dass die TelematikID oder die KIM Adresse anzugeben ist? 
Als regex oder eigenes URL konstrukt bauen, was dann eben nur KIM Adressen akzeptiert?

Reply:
Habe das Thema nochmal mit Hendrik diskutiert, so wie sie es im VSDM Kontext umgesetzt haben und was deren Konzept ist. Die Kim Adresse verpflichtend anzugeben hätte den Nachteil, das man den Inhalt der Nachricht/Bundle an das Medium hängt. Besser wäre es optional angeben zu können, an wen die Antwort gesendet werden soll (für den Falls, dass als Kim Nachricht) Ich versuche das mal umzusetzen.

Reply:
Die Information der wie der Empfängers zu erreichen ist, muss in der GEM_PR_ERP_MEDREQ_RequestingOrganization verpflichtend angeben werden.
Der Header sollte erstmal nur Auskunft geben müssen, wer der Sender ist.